tests/fixtures/app/typescript-embroider/tsconfig.json

```diff
@@ -12,18 +12,18 @@
     },
     "types": [
       "ember-source/types",
-      "ember-data/unstable-preview-types",
-      "@ember-data/store/unstable-preview-types",
-      "@ember-data/adapter/unstable-preview-types",
-      "@ember-data/graph/unstable-preview-types",
-      "@ember-data/json-api/unstable-preview-types",
-      "@ember-data/legacy-compat/unstable-preview-types",
-      "@ember-data/request/unstable-preview-types",
-      "@ember-data/request-utils/unstable-preview-types",
-      "@ember-data/model/unstable-preview-types",
-      "@ember-data/serializer//unstable-preview-types",
-      "@ember-data/tracking/unstable-preview-types",
-      "@warp-drive/core-types/unstable-preview-types"
+      "./node_modules/ember-data/unstable-preview-types",
```

Comment:
Whys this path required?

Reply:
Without it TS throws errors, more info in the [Discord thread](https://discord.com/channels/480462759797063690/1290318997040005234).

Reply:
It would be nice if there were tests that run the linters against the generated blueprints so this would have been caught in CI. Not sure how hard that would be to set up.